htdocs/comm/propal/card.php

```diff
@@ -1238,7 +1238,7 @@
 
 			// Prepare a price equivalent for minimum price check
 			$pu_equivalent = $pu_ht;
-			$pu_equivalent_ttc = $pu_ttc;
+			//$pu_equivalent_ttc = $pu_ttc;
```

Comment:
I see no reason to disable this line as this is just an assignation.
If as a business rule, we decide to check minimum ONLY for price without vat, we should disable the code that do the comparison but not this assignation.